lib/index.js

```diff
@@ -174,7 +175,7 @@ function mergePath(type, into, pathRoot, rootPolicy, policy) {
     }
 
     // otherwise we need to merge up manually
-    rootPolicy[into][id] = rootPolicy[type][id].concat(policy[type][id]);
+    rootPolicy[into][id] = rootPolicy[into][id].concat(policy[type][id]);
```

Comment:
This was failing because ignores are merged `into` a value called `"suggest"`, but the `rootPolicy.ignore` didn't contain the `id` that was originally referenced, so the `.concat` error would throw.

Now it correctly merges from the source property (this can be `into` = `(suggest | ignore | patch)`).